network/p2p/peerManager_test.go

```diff
@@ -142,7 +142,7 @@ func (suite *PeerManagerTestSuite) TestPeriodicPeerUpdate() {
 		}
 	}).Return(nil)
 
-	peerUpdateInterval := 5 * time.Millisecond
+	peerUpdateInterval := 10 * time.Millisecond
```

Comment:
Is this increase in update interval to decrease the chance of timing out?

Reply:
Just to decrease the chance of all the steps in between adding unintended latency which puts us over the time limit.